src/popover/popover.jsx

```diff
@@ -162,12 +162,10 @@ const Popover = React.createClass({
         if (nextProps.animated) {
           this.setState({closing: true});
           this._timeout = setTimeout(() => {
```

Comment:
Sure! I really wanted to before but I was trying to minimize line changes so it was easier to review the approaches I was taking. :smile: